src/richie/apps/courses/templates/courses/plugins/organization.html

```diff
@@ -0,0 +1,18 @@
+{% load cms_tags i18n thumbnail %}
+{% spaceless %}
+<div class="organization-plugin">
+  {% if instance.organization.logo %}
+    <img
+      class="organization-plugin__logo"
+      src="{{ instance.organization.logo.url }}"
```

Comment:
I think you need to wrap this with a thumbnail template tag to generate  a thumbnail in the size you want. People may upload it in various dimensions...

Reply:
I tried this:
```html
{% load cms_tags i18n thumbnail %}
{% spaceless %}
<div class="organization-plugin">
  {% if instance.organization.logo %}
    <img
      class="organization-plugin__logo"
      src="{{ instance.organization.logo.url }}"
      alt="{% blocktrans with title=instance.organization.extended_object.get_title|capfirst %}{{ title }} logo{% endblocktrans %}">
    {% thumbnail instance.organization.logo "100x100" crop="center" as im%}
      <img
      class="organization-plugin__logo"
      src="{{ im.url }}"
      alt="{% blocktrans with title=instance.organization.extended_object.get_title|capfirst %}{{ title }} logo{% endblocktrans %}">
    {% endthumbnail %}
  {% endif %}
```
But it gives me : 
```
Error during template rendering
In template /app/src/richie/apps/courses/templates/courses/plugins/organization.html, error at line 14

Invalid block tag on line 14: 'endthumbnail', expected 'elif', 'else' or 'endif'. Did you forget to register or load this tag?
```
Any pointers on what i'm doing wrong? I am using this as reference: http://sorl-thumbnail.readthedocs.io/en/latest/examples.html

Reply:
We don't use sorl thumbnail but easy-thumbnail : http://easy-thumbnails.readthedocs.io/en/stable/usage/#easy_thumbnails.templatetags.thumbnail.thumbnail

Reply:
There is an example here: https://github.com/openfun/richie/blob/ce6ca105d43ab48f849a14e0a52d4e4b53a68dbd/src/richie/plugins/large_banner/templates/large_banner.html

Actually, we should move this logo to a placeholder like it's done on subjects WDYT?  This is a relique from the first days of Richie...

Reply:
That would be nice. Add the change here or make a different pull request?